docs/output.md

```diff
@@ -156,9 +157,7 @@ When graphs are computed and identified, their ID names are added back to the ed
 
 The graph command has the option to recover components (technical multiplets) into smaller
 components using community detection to find and remove problematic edges
-(see `--multiplet_recovery`). These new component IDs are then stored in the "component" column. The information to keep track of the original and
-newly recovered components are stored in a file (components_recovered.csv).
-This file is not included in the output folder by default, but can be included by passing `--save_recovered_components`.
+(see `--multiplet_recovery`). These new component IDs are then stored in the "component" column.
```

Comment:
stored in column where ?

Reply:
These new components are added inside the pixeldataset which consists in part of an embedded anndata object. I find the whole last sentence a bit confusing so I have just removed it. Extensive documentation on how to deal with the pipeline output files is provided in the [Pixelgen data analysis tutorials](https://software.pixelgen.com/mpx-analysis/latest/introduction/).